templates/apis/crd.go.tpl

```diff
@@ -40,7 +40,7 @@ type {{ .CRD.Kind }}Status struct {
 	// +kubebuilder:validation:Optional
 	Conditions []*ackv1alpha1.Condition `json:"conditions"`
 	{{- range $fieldName, $field := .CRD.StatusFields }}
-	{{- if $field.ShapeRef.Documentation }}
+	{{- if and ($field.ShapeRef) ($field.ShapeRef.Documentation) }}
```

Comment:
Not relevant to this PR? Rebasing issue?

Reply:
yeah, looks like a rebasing issue.

Reply:
FIxed